### DISCUSSION TITLE
Welcome to flipp_pomodoro Discussions!

### DISCUSSION THREAD

**Discussion by Th3Un1q3** [Th3Un1q3/flipp_pomodoro#15] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Folzi99** [comment]
Just wanted to say thank you for creating this tool. 

I've been using it as a distraction free way to keep focused while studying. 

I am very much a newbie to programing/electronics. Learning how many ways the flipper can be used outside of the original intended purposes is so cool. 

Thanks again!

😊

**Reply by Th3Un1q3** [reply]
@Folzi99 Thank you so much for your encouraging words! As the developer of the app, it's incredibly rewarding to hear how much you're benefitting from it. It makes all the hard work worth it!

Don't worry about being new to programming and electronics - everyone has to start somewhere. It's fantastic to see your enthusiasm for learning and experimenting with the flipper.

Thanks once again for taking the time to share your experience. Happy studying!
